install/helm/agones/values.yaml

```diff
@@ -51,21 +51,14 @@ agones:
       # requests:
       #   cpu: 1
       #   memory: 256Mi
+    requireDedicatedNode: false
```

Comment:
I'm trying to think of a use case where someone would set one of these to true but not all of them. While having fine-grained settings can be nice, it also means you have to remember to set all 3 variables instead of just a single one when you want the required affinity. 

WDYT about having a single variable vs. 3 variables?

@zmerlynn - do you have any preference on this?

Reply:
Now I'm leaning more to have a single flag as start to avoid overengineering. We can always provide more config later.